hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlTestSupport.java

```diff
@@ -104,6 +104,9 @@ public static void createTable(String tableName) throws SQLException {
         createTable(tableName, "id INT PRIMARY KEY", "name VARCHAR(100)");
     }
 
+    protected void createTableWithQuotation(String tableName) throws SQLException {
```

Comment:
We should do the opposite here
- `createTable` should take the table name and quote it

And we should have `createTableNoQuote` or similar which will expect quoted table name.

The resulting changeset will be much smaller.